AlicerceTests/ServiceLocatorTests.swift

```diff
@@ -0,0 +1,695 @@
+//
+//  ServiceLocatorTests.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 26/01/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+
+@testable import Alicerce
+
+class ServiceLocatorTests : XCTestCase {
+    
+    var serviceLocator = ServiceLocator()
+    
+    override func setUp() {
+        super.setUp()
+        
+        serviceLocator = ServiceLocator()
+    }
+
+    // MARK: - Register (normal)
+    // MARK: Success
+    
+    func testRegister_UsingTypeInference_ShouldRegisterServices() {
+        
+        let testServiceString = "Test Register Service"
+        let testServiceDouble = 1.0
+        let testServiceInt = 1
+        
+        do {
+            
+            // Register services
+            try serviceLocator.register(service: testServiceString)
+            try serviceLocator.register(service: testServiceDouble)
+            try serviceLocator.register(service: testServiceInt)
+            
+            // Get registered services
+            let rStringService: String = try serviceLocator.get()
+            let rDoubleService: Double = try serviceLocator.get()
+            let rIntService: Int = try serviceLocator.get()
+            
+            // Validate registered services
+            XCTAssertEqual(rStringService, testServiceString)
+            XCTAssertEqual(rDoubleService, testServiceDouble)
+            XCTAssertEqual(rIntService, testServiceInt)
+            
+        } catch let error {
```

Comment:
I don't think you need to declare the error variable here, as its now added by default.